staging/src/k8s.io/apiserver/pkg/registry/generic/rest/proxy.go

```diff
@@ -78,7 +78,7 @@ func NewUpgradeAwareProxyHandler(location *url.URL, transport http.RoundTripper,
 
 // ServeHTTP handles the proxy request
 func (h *UpgradeAwareProxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
-	if len(h.Location.Scheme) == 0 {
+	if h.Location.Scheme == "" {
```

Comment:
Switched to match the style guide. I think its important to be consistent with the style guide. At some point we may want to think about checking empty string vs empty in the style guide. Empty string makes the expected type clear and will blow up if the Dev's assumption of type is broken.